contracts/token/ERC20/MintableToken.sol

```diff
@@ -7,7 +7,7 @@ import "../../ownership/Ownable.sol";
 /**
  * @title Mintable token
  * @dev Simple ERC20 Token example, with mintable token creation
- * @dev Issue: * https://github.com/OpenZeppelin/openzeppelin-solidity/issues/120
+ * Issue: * https://github.com/OpenZeppelin/openzeppelin-solidity/issues/120
```

Comment:
Can we actually just remove this line? It's not useful anymore.

Reply:
Done.